cylc/flow/network/schema.py

```diff
@@ -1725,7 +1725,7 @@ class Meta:
     class Arguments:
         workflows = List(WorkflowID, required=True)
         mode = WorkflowStopMode(
-            default_value=WorkflowStopMode.Clean.value
+            default_value=WorkflowStopMode.Clean.name
```

Comment:
Argh, with this the UI stop button works (also when you use the edit button, "Clean" is pre-populated in the mode dropdown), but `cylc stop foo` results in
```
WARNING - Invalid stop mode: 'Clean'
WARNING - Command failed: stop(mode=Clean)
```
But with `default_value=WorkflowStopMode.Clean.value`, `cylc stop foo` works whereas the stop button in the UI has the 400 error!